README.md

```diff
@@ -71,65 +72,96 @@ Provides the following HTTP methods for `/load-test` endpoint:
 
  If you prefer to use Postman you can also import [openapi.json](openapi.json) file into Postman to create a new collection.
 
-### Kangal controller
-The general name for several Kubernetes controllers created to manage all the aspects of the performance testing process.
- - LoadTest controller  
- - Backend jobs controller
- 
-## To start using Kangal
```

Comment:
I think we should keep this title and put your content under it

Reply:
This section was literally pointing to `docs/`, I just renamed to `Documentation` and reduced the number of linkings